### DISCUSSION TITLE
How to Update Template

### DISCUSSION THREAD

**Discussion by adamrcooper** [excalidraw/excalidraw-vscode#49] [Q&A] [answered]
Hello - I'm not familiar with coding. I just started using Excalidraw as part of Obsidian. Is there a way to update the template with my current settings? Thank you. - Adam

**Comment by pomdtr** [comment, marked as answer]
Hey, you should ask these questions directly in the obsidian plugin repository : 
https://github.com/zsviczian/obsidian-excalidraw-plugin

**Comment by adamrcooper** [comment]
Thank you for the help. :)
